src/ManagedServices/ManagedServices/Commands/GetAzureRmManagedServicesAssignment.cs

```diff
@@ -27,44 +27,29 @@ namespace Microsoft.Azure.PowerShell.Cmdlets.ManagedServices.Commands
     public class GetAzureRmManagedServicesAssignment : ManagedServicesCmdletBase
     {
         protected const string DefaultParameterSet = "Default";
-        protected const string ByResourceIdParameterSet = "ByResourceId";
-        protected const string ByIdParameterSet = "ById";
+        protected const string ByNameParameterSet = "ByName";
 
         [Parameter(Mandatory = false, ParameterSetName = DefaultParameterSet, HelpMessage = "The scope where the registration assignment is created.")]
-        [Parameter(Mandatory = false, ParameterSetName = ByIdParameterSet, HelpMessage = "The scope where the registration assignment is created.")]
+        [Parameter(Mandatory = false, ParameterSetName = ByNameParameterSet, HelpMessage = "The scope where the registration assignment is created.")]
         [ScopeCompleter]
         public string Scope { get; set; }
 
-        [Parameter(Mandatory = true, ParameterSetName = ByIdParameterSet, HelpMessage = "The Registration Assignment identifier.")]
+        [Parameter(Mandatory = true, ParameterSetName = ByNameParameterSet, HelpMessage = "The unique name of the Registration Assignment.")]
         [ValidateNotNullOrEmpty]
-        public string Id { get; set; }
-
-        [Parameter(
-            Mandatory = true,
-            ValueFromPipelineByPropertyName = true,
-            ParameterSetName = ByResourceIdParameterSet,
-            HelpMessage = "The fully qualified resource id of registration assignment.")]
-        [ValidateNotNullOrEmpty]
-        public string ResourceId { get; set; }
```

Comment:
removing mandatory parameter is breaking change and is not allowed in non-breaking change release